EnglishTranslation-master/SKILL.tsv

```diff
@@ -4582,13 +4582,13 @@ SKILL_20150714_004581	Increases the damage dealt on an enemy with [Barbed Arrow]
 SKILL_20150714_004582	Increases the damage dealt on an enemy with [Crossfire] by 1% per attribute level.
 SKILL_20150714_004583	Increases the damage dealt on an enemy with [Magic Arrow] by 1% per attribute level.
 SKILL_20150714_004584	Increases the damage dealt on an enemy with [Divine Machine Arrow] by 1% per attribute level.
-SKILL_20150714_004585	Increases the time duration of the [Bleeding] status when an enemy is hit by [Broad Head] by 2 seconds per attribute level.
+SKILL_20150714_004585	Increases the duration of the [Bleeding] status when an enemy is hit by [Broad Head] by 2 seconds per attribute level.
 SKILL_20150714_004586	Bodkin Point: Decreased Defense
-SKILL_20150714_004587	[Reduced DEF] of [Bodkin Point] will be reduced by 2% per Attribute level.
-SKILL_20150714_004588	The enemies that are shot by [Magic Arrow] will fall into [Silence] with a certain probabillity. The probability will increase by 10% per Attribute level.
+SKILL_20150714_004587	The [Reduced DEF] effect of [Bodkin Point] will be increased by 2% per Attribute level.
```

Comment:
`Reduces the [Decreased Defense] effect from [Bodkin Point] by 2% per attribute level.`